src/main/java/com/extendedclip/papi/expansion/essentials/EssentialsExpansion.java

```diff
@@ -197,6 +198,19 @@ public String onRequest(OfflinePlayer player, @NotNull String identifier) {
 
         if (player == null) return "";
 
+        if (identifier.startsWith("tp_cooldown")) {
```

Comment:
If the placeholder is just `%essentials_tp_cooldown%` (like your PR title suggests) could you just use `equals` here and not `startsWith`

Reply:
Sure, I changed it, thanks.